src/BarcodeScannerForPhoneGap/widget/BarcodeScannerForPhoneGap.js

```diff
@@ -55,6 +56,7 @@ define([
             if (!output.cancelled && output.text && output.text.length > 0) {
                 this._obj.set(this.attributeName, output.text);
                 this._executeMicroflow();
+                this._executeNanoflow();
```

Comment:
Rename the _executeMicroflow() to be generic maybe executeAction and handle both events from in there

Reply:
Done